views/partials/navigation.hbs

```diff
@@ -7,11 +7,13 @@
     <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
         <div class="navbar-nav">
             <a class="nav-item nav-link {{#if dashboard}} active {{/if}}" href="/">Dashboard{{#if dashboard}}<span class="sr-only">(current)</span>{{/if}}</a>
-            <a class="nav-item nav-link {{#if login}} active {{/if}}" href="login">Login{{#if login}}<span class="sr-only">(current)</span>{{/if}}</a>
-            <a class="nav-item nav-link {{#if user}} active {{/if}}" href="user">User{{#if user}}<span class="sr-only">(current)</span>{{/if}}</a>
-            <a class="nav-item nav-link {{#if product_catalog}} active {{/if}}" href="product_catalog">Product Catalog{{#if product_catalog}}<span class="sr-only">(current)</span>{{/if}}</a>
-            <a class="nav-item nav-link {{#if inventory}} active {{/if}}" href="inventory">Manage Inventory{{#if inventory}}<span class="sr-only">(current)</span>{{/if}}</a>
             <a class="nav-item nav-link {{#if transaction}} active {{/if}}" href="/transaction">New Transaction{{#if transaction}}<span class="sr-only">(current)</span>{{/if}}</a>
+            <a class="nav-item nav-link {{#if inventory}} active {{/if}}" href="inventory">Manage Inventory{{#if inventory}}<span class="sr-only">(current)</span>{{/if}}</a>
+            <a class="nav-item nav-link {{#if product_catalog}} active {{/if}}" href="product_catalog">Product Catalog{{#if product_catalog}}<span class="sr-only">(current)</span>{{/if}}</a>
+            <a class="nav-item nav-link {{#if user}} active {{/if}}" href="user">Users{{#if user}}<span class="sr-only">(current)</span>{{/if}}</a>
         </div>
     </div>
+    <div class="navbar-nav">
+        <a class="nav-item nav-link" href="/logout">Logout</a>
+    </div>

```

Comment:
If you have any suggestions for better way to structure, please let me know. Ideally I'd like the "logout" link to be in the same hamburger menu when the nav bar collapses...but if I added the `a` tag to the sibling `div` above, I couldn't figure out how to right-align.